chia/wallet/cat_wallet/cat_utils.py

```diff
@@ -82,7 +83,8 @@ def subtotals_for_deltas(deltas) -> List[int]:
 def next_info_for_spendable_cat(spendable_cat: SpendableCAT) -> Program:
     c = spendable_cat.coin
     list = [c.parent_coin_info, spendable_cat.inner_puzzle.get_tree_hash(), c.amount]
-    return Program.to(list)
+    ret: Program = Program.to(list)
+    return ret
```

Comment:
I don't like re-hinting return values in general, but especially not when they hide the underlying issue.
```suggestion
    # ignoring hint error here for:
    # https://github.com/Chia-Network/clvm/pull/102
    return Program.to(list)  # type: ignore[no-any-return]
```